pkg/routeagent_driver/handlers/kubeproxy/kp_packetfilter.go

```diff
@@ -153,9 +153,12 @@ func (kp *SyncHandler) Init(_ context.Context) error {
 		// Configure CNI Specific changes
 		kp.cniIface = cniIface
 
-		err := kp.netLink.EnableLooseModeReversePathFilter(kp.cniIface.Name)
-		if err != nil {
-			return errors.Wrap(err, "error enabling loose mode")
+		// rp_filter isn't supported for IPv6.
+		if kp.ipFamily == k8snet.IPv4 {
+			err := kp.netLink.EnableLooseModeReversePathFilter(kp.cniIface.Name)
```

Comment:
We could add the IPFamily to `EnableLooseModeReversePathFilter` as well and handle skipping IPv6 in one place.